Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessClusterManager.java

```diff
@@ -172,8 +193,10 @@ public ClusterContainer refresh() {
 
         // Loop subscriptions to get all accounts
         return Observable
-                .fromCallable(() -> getAzureManager().getSubscriptionManager().getSubscriptionDetails())
-                .flatMap(Observable::from)             // Get Subscription details one by one
+                .fromCallable(() -> getAzureManager().getSubscriptionManager().getSubscriptionIdToSubscriptionDetailsMap())
+                .flatMap(sidToDetails -> Observable.from(getAzureManager().getSubscriptionManager().getAccountSidList())
+                                                    // Handle account one by one
+                                                   .map(sidToDetails::get))
```

Comment:
Ideally, this should be in common function of SparkServerless and other cluster type.

Reply:
fix it.